packages/components/src/menu-group/index.js

```diff
@@ -33,11 +28,12 @@ export function MenuGroup( {
 	return (
 		<div className={ classNames }>
 			{ label &&
-				<div className="components-menu-group__label" id={ labelId }>{ label }</div>
+				<div
+					className="components-menu-group__label" id={ labelId } aria-hidden="true">{ label }</div>
```

Comment:
Unsure if by the indented newline you had planned to split this across multiple lines? Seems like it could be an improvement for line length readability.

```suggestion
					className="components-menu-group__label"
					id={ labelId }
					aria-hidden="true"
				>
					{ label }
				</div>
```

Reply:
👍